docs/Servo.md

```diff
@@ -0,0 +1,72 @@
+#Phidget Spatial Sensor
+The PhidgetServo library makes for intuitive and lightning fast development without any compromise. For a quick start into your Spatial project see this [basic PhidgetServo example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/servoMotor.js).
+
+##Methods
+
+|Method call|Parameters|Description|
+|---|---|---|
+|[phidget.connect](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|[__phidget__.params object](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#connecting--phidgetparams)|Connects the PhidgetServo|
+|[phidget.quit](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/Phidget.md#methods)|N/A |This method requests a disconnect from the phidget device.  The disconnected event will be dispatched when the connection has been successfully disconnected.|
+|whenReady|function|This executes a function when the Phidget Servo is ready to be used.|
+|observe|function|Used for asynchronously observing changes to the Phidget Servo. This behaves much like the JS [Object.observe](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/observe), however you only pass the handler, not the object or accept list. All of this is handled by the Phidget Framework.|
+
+
+##Data
+
+|Key|Data Type|Writable|Description|
+|---|---|---|---|
+|type|string|no|`'PhidgetServo'`|
+|positionMaxLimit|number|no|Global upper position Limit in `degrees`|
+|positionMinLimit|number|no|Global lower position Limit in `degrees`|
+|numberOfMotors|number|no|Number of motors|
+|engaged|array|yes|Power state of motors: `1` = on, `0` = off|
+|positions|array|yes|Position of each motor in degrees|
+|positionMax|array|no|Max position of each servo in degrees|
+|positionMin|array|no|Min position of each servo in degrees|
+|[servoParameters](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/ServoParameters.md)|array|no|ServoParams for each motor [see here](https://github.com/RIAEvangelist/node-phidget-API/blob/master/docs/ServoParameters.md)|
```

Comment:
Also, can you just double check that the positions Min and Max etc are in degrees not us?